backend/apps/owasp/api/internal/queries/certificate.py

```
@@ -0,0 +1,43 @@
+
+from apps.nest.api.internal.permissions import IsAuthenticated
+from apps.owasp.api.internal.nodes.certificate import CertificateNode
+from apps.owasp.models.crp.certificate import Certificate
+
+
+@strawberry.type
+class CertificateQuery:
+    """Certificate queries."""
+
+    @strawberry_django.field
+    def certificate(self, id: uuid.UUID) -> CertificateNode | None:
```

Ruff (0.15.17)

[error] Function argument `id` is shadowing a Python builtin

(A002)